finite_state_sdk/queries.py

```diff
@@ -502,6 +502,7 @@ def _create_GET_FINDINGS_VARIABLES(asset_version_id=None, category=None, cve_id=
             cvssBaseMetricV3 {
                 cvssv3 {
                     baseScore
+                    vectorString
```

Comment:
Do we need to make any model changes for this to show up or is that dynamic?

Reply:
it's dynamic. No model changes is needed